x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -323,10 +323,12 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
 
   public start(core: CoreStart, plugins: StartPlugins) {
     KibanaServices.init({ ...core, ...plugins, kibanaVersion: this.kibanaVersion });
-    plugins.ingestManager.registerPackageConfigComponent(
-      'endpoint',
-      ConfigureEndpointPackageConfig
-    );
+    if (plugins.ingestManager) {
```

Comment:
I think the problem with tests failing without this conditional should have been caught by typescript. Is it because we should have have also set the `ingestManager` `StartPlugins` type as optional? https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/types.ts#L35

@paul-tavares @XavierM @kevinlog thoughts?

Reply:
@peluja1012 yeah, that sounds like it needs to be set to optional in the Type